packages/myst-to-ipynb/tests/run.spec.ts

```diff
@@ -3,11 +3,11 @@ import fs from 'node:fs';
 import path from 'node:path';
 import yaml from 'js-yaml';
 import { unified } from 'unified';
-import mystToMd from '../src';
+import writeIpynb from '../src';
 
 type TestCase = {
   title: string;
-  markdown: string;
+  ipynb: string;
```

Comment:
We _probably_ don't care about the _exact_ text representation of the `ipynb` file on disk. Could you change the `ipynb` entry to be a `Record<string, any>?`

Then, you'd need to update the tests to be

````yaml
- title: link reference
  mdast:
   type: root
   children:
     - type: text
       value: markdown
  ipynb: 
    cells:
      - 
````
instead of 

````yaml
- title: link reference
  mdast:
   type: root
   children:
     - type: text
       value: markdown
  ipynb: |-
      {
        "cells": [
````

This would enable us to write the regression test in YAML (less verbose) _and_ to choose to omit parts of the output that we don't want to test (e.g. if there were timestamps)